{
  "error": "LiDAR Malfunction detected in database",
  "faultySensor": "LiDAR1",
  "lastLiDarWorkerTrackersFrame": {
    "LiDAR1": {
      "time": 4,
      "trackedObjects": [
        {
          "id": "Wall_1",
          "time": 2,
          "description": "Wall",
          "coordinates": [
            {
              "x": 0.1176,
              "y": 3.6969
            },
            {
              "x": 0.11362,
              "y": 3.6039
            }
          ]
        }
      ]
    }
  },
  "statistics": {
    "systemRuntime": 4,
    "numDetectedObjects": 5,
    "numTrackedObjects": 1,
    "numLandmarks": 1,
    "landMarks": {
      "Wall_1": {
        "id": "Wall_1",
        "description": "Wall",
        "coordinates": [
          {
            "x": 0.4908953928360109,
            "y": 0.12055924334093296
          },
          {
            "x": 0.3978103367934267,
            "y": 0.12044636487120144
          }
        ]
      }
    }
  }
}